### PR TITLE
add new terms Diandrous and monandrous

### LINKED ISSUE #257: NTR: Diandrous and monandrous
This will be tricky because diandrous and monandrous mean different things for plants and animals. How should we differentiate in the label?

Preferred Label
Diandrous
Monandrous

Definition
Diandrous (plant): having two stamens
Monandrous (plant): having one stamen
Diandrous (animal): possessing two different types of males
Monandrous (animal): having only one type of male

Parent Class
The plant terms should be subclasses of 'phenotypic sex'
the animal terms should be subclasses of 'mixed sex'

Attribution
http://orcid.org/0000-0002-2908-3327

Notes
There is also diandry: fertilization of an egg by two sperm.

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
closes #257

### CODE REVIEW

**Comment by ramonawalls** [comment]
I would like to add to this PR by including terms for diandrous and monandrous. Even though they were requested in PCO (see PopulationAndCommunityOntology/pco#69) because they are population qualities, I think they should be created in PATO then imported into PCO.

**Comment by ramonawalls** [comment]
I will check out this branch and make the changes, assuming I can get the github method right.

**Comment by nicolevasilevsky** [comment]
@ramonawalls if you have any issues, I'm around right now and can (try to) help

**Comment by ramonawalls** [comment]
@diatomsRcool, I cannot add you as a reviewer, because you are not a member of this repo, but please do review this PR.

**Comment by diatomsRcool** [comment]
Looks good to me

**Comment by cmungall** [comment]
@nicolevasilevsky can you fix the conflicts?

Also this PR seems to have more changes than is advertised in the title

**Comment by matentzn** [comment]
@nicolevasilevsky please review carefully the isas before merging, because of all the hack-slashing i needed to do

**Comment by nicolevasilevsky** [comment]
will do, thanks for your help @matentzn

File: src/ontology/pato-edit.obo [hunk 1]
```diff
@@ -20612,6 +20613,54 @@ synonym: "skin redness" EXACT []
...
+
+[Term]
+id: PATO:0040051
+name: monandrous
+def: "A phenotypic sex quality inhering in a  plant organism or a population of plants in which the flowers have only one stamen." [http://orcid.org/0000-0002-2908-3327, https://orcid.org/0000-0001-8815-0078]
+is_a: PATO:0001894 ! phenotypic sex
+created_by: http://orcid.org/0000-0001-5208-3432
+
+[Term]
+id: PATO:0040052
```
Comment (nicolevasilevsky) [inline src/ontology/pato-edit.obo:20631]: when I open the OWL file, the ID for this term is PATO_0070037

File: src/ontology/pato-edit.obo [hunk 2]
```diff
@@ -20612,6 +20613,54 @@ synonym: "skin redness" EXACT []
...
+name: protogynous
+def: "A sequential hermaphroditic quality whereby organisms start out as female and change to male at a later life stage." [https://en.wikipedia.org/wiki/Sequential_hermaphroditism] {comment="http://orcid.org/0000-0002-2908-3327"}
+synonym: "protogyny" EXACT []
+is_a: PATO:0070000 ! sequential hermaphrodite
+created_by: https://orcid.org/0000-0001-8815-0078
+
+[Term]
+id: PATO:0040053
```
Comment (nicolevasilevsky) [inline src/ontology/pato-edit.obo:20639]: the ID for this term is PATO_0070038 in the OWL file

File: src/ontology/pato-edit.obo [hunk 3]
```diff
@@ -20612,6 +20613,54 @@ synonym: "skin redness" EXACT []
...
+name: protandrous
+def: "A sequential hermaphroditic quality whereby organisms start out as male and change to female at a later life stage." [http://orcid.org/0000-0002-2908-3327, https://en.wikipedia.org/wiki/Sequential_hermaphroditism]
+synonym: "protandry" EXACT []
+is_a: PATO:0070000 ! sequential hermaphrodite
+created_by: https://orcid.org/0000-0001-8815-0078
+
+[Term]
+id: PATO:0040054
```
Comment (nicolevasilevsky) [inline src/ontology/pato-edit.obo:20647]: in the OWL file, this ID is PATO_0070036

and the parent is 'neurogliaform morphology' (which seems wrong)

**Comment by nicolevasilevsky** [comment]
@matentzn there seem to be lots of issues with this - I think it's probably easier to just close this and start over. I can do so, once you give me the go ahead.

File: src/ontology/pato-edit.obo (hunk 2 above)
Comment (matentzn) [reply src/ontology/pato-edit.obo:20639]: What OWL file?

Comment (matentzn) [reply src/ontology/pato-edit.obo:20639]: What OWL file?

Comment (nicolevasilevsky) [reply src/ontology/pato-edit.obo:20639]: the pato-edit.owl file

Comment (nicolevasilevsky) [reply src/ontology/pato-edit.obo:20639]: Oh, I guess it is pato-edit.obo, that is what I meant, obo, not owl

Comment (nicolevasilevsky) [reply src/ontology/pato-edit.obo:20639]: image

File: src/ontology/pato-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/pato-edit.obo:20631]: image

Comment (matentzn) [reply src/ontology/pato-edit.obo:20631]: I dont know, something seems really wrong here.. Maybe lets do this together in our face2face on Thursday. The only thing that can be wrong is the subclassOf axioms - the terms should be correct.

**Comment by matentzn** [comment]
Lets discuss at next meeting.

**Comment by nicolevasilevsky** [comment]
talked to @matentzn - I will redo this

**Comment by nicolevasilevsky** [comment]
recreated this PR, closing this.

**Pull request closed without merging**